.github/workflows/approved-label.yml

```diff
@@ -12,3 +12,5 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         ADD_LABEL: "approved"
         REMOVE_LABEL: ""
+    permissions:
+      pull_request: write
```

Comment:
Should be closer to the start of the file

Reply:
added after on block